cpp/Platform.Timestamps/CommonEraClock.h

```diff
@@ -0,0 +1,65 @@
+#ifndef PLATFORM_TIMESTAMPS_COMMON_ERA_CLOCK
+#define PLATFORM_TIMESTAMPS_COMMON_ERA_CLOCK
+
+#include <chrono>
+
+namespace Platform::Timestamps
+{
+    struct common_era_clock
+    {
+        using ticks = std::chrono::duration<std::uint64_t, std::ratio_multiply<std::nano, std::ratio<100>>>;
+        using duration = ticks;
+        using rep = duration::rep;
+        using period = duration::period;
+        using time_point = std::chrono::time_point<common_era_clock>;
+        static constexpr bool is_steady = false;
```

Comment:
We might add empty line between types and constants?

Reply:
No need, as long as the file have consistent style.